vcbuild.bat

```diff
@@ -381,7 +381,8 @@ if defined msbuild_args set "extra_msbuild_args=%extra_msbuild_args% %msbuild_ar
 set UseMultiToolTask=True
 set EnforceProcessCountAcrossBuilds=True
 set MultiProcMaxCount=%NUMBER_OF_PROCESSORS%
-msbuild node.sln %msbcpu% /t:%target% /p:Configuration=%config% /p:Platform=%msbplatform% /clp:NoItemAndPropertyList;Verbosity=minimal /nologo %extra_msbuild_args%
+msbuild node.sln %msbcpu% /t:%target% /p:Configuration=%config% /p:Platform=%msbplatform% /clp:NoItemAndPropertyList;Verbosity=minimal /nologo %extra_msbuild_args% /guard:cf
```

Comment:
This is failing because `/guard` is a `cl` option, not `msbuild`. *I think* just adding `<ControlFlowGuard>Guard</ControlFlowGuard>` to the list of options enclosed by the `<ClCompile>` tags in [`tools/msvs/msi/custom_actions.vcxproj`](https://github.com/nodejs/node/blob/HEAD/tools/msvs/msi/custom_actions.vcxproj) should fix this.

Reply:
Is this file used to compile `node.exe` ?

Reply:
No, AFAIK it's only used to compile the custom action for the installer. We'd probably need to find an appropriate place in the gyp files.